pkg/kmodule/kmodule_linux.go

```diff
@@ -33,17 +33,52 @@ const (
 	MODULE_INIT_IGNORE_VERMAGIC = 0x2
 )
 
+type LinuxLoader struct {
+	modules string
+}
+
 // Init loads the kernel module given by image with the given options.
-func Init(image []byte, opts string) error {
+func (l *LinuxLoader) Init(image []byte, opts string) error {
 	return unix.InitModule(image, opts)
 }
 
+func (l *LinuxLoader) List(w io.Writer) error {
+	file, err := os.Open(l.modules)
+	if err != nil {
+		return err
+	}
+	defer file.Close()
+
+	if _, err := fmt.Fprintln(w, "Module                  Size  Used by"); err != nil {
```

Comment:
instead of rolling our own table and columns here, maybe [`text.tabwriter`](https://pkg.go.dev/text/tabwriter) can be helpful?  

Example here: https://stackoverflow.com/questions/36101534/the-efficient-way-to-print-a-table-in-go

Reply:
I hate understood tabwriter to be deprecated, let me go look ...

Reply:
I see that it's frozen, but does that mean deprecated? That is a legitimate question, the answer is unclear to me.  IIUC it's part of the core library and that's guaranteed to not break in v 1... so even if it's not accepting new features it should continue to work indefinitely?

Reply:
er, "I had" ... 
This actually was designed to align to what you get from /proc/modules. It's a bit fragile: if proc/modules alignment changes, well, this won't be right.
I don't know the best approach. tabwriter seems a bit overkill, since the rest of the text is aligned by the kernel ...

Reply:
I always took frozen to mean "we won't remove it but we won't fix or improve it" -- which to me means "not for  new code"

Reply:
I went with tab writer. see kmodule.Pretty